molgenis/molgenis-emx2
# feat: add collection and subcollection count to biobank and network details

What are the main changes you did:
-closes #3663 

how to test:
- Go to https://preview-emx2-pr-4032.dev.molgenis.org
- upload bbrmi dataset
- go to directory app
- go to a biobank that has subcollections 
- See that both the collection count and the subcollection count are shown.

- Go to a network that has subcollection 
- See that the collection count is now split into sub and parent collections
